From: Dermot Bradley <dermot_bradley@yahoo.com>
Date: Wed, 28 Oct 2020 20:12 +0000
Subject: [PATCH] cloud-utils: Minor documentation and testcase corrections

cloud-publish-ubuntu relies on 2 scripts that were removed and so in
non-functional without them. Remove it from README and tox configuration.

This change is part of upstream submitted PR 20.

---

diff -aur a/README.md b/README.md
--- a/README.md
+++ b/README.md
@@ -13,12 +13,11 @@
 
 Red Hat based distributions offer the `cloud-utils` package with a subset of binaries and `cloud-utils-growpart` for growpart.
 
-The package is also available in Apline and Arch as `cloud-utils`.
+The package is also available in Alpine and Arch as `cloud-utils`.
 
 ## Usage
 
 * `cloud-localds` - create a disk for cloud-init to utilize the nocloud datasource
-* `cloud-publish-ubuntu` - import a cloud image to your local cloud
 * `ec2metadata` - query and display AWS EC2 metadata
 * `growpart` - rewrite partition table so that partition takes up all the space it can
 * `mount-image-callback` - mount a file to a temporary mount point and then invoke the provided cmd with args
diff -aur a/tox.ini b/tox.ini
--- a/tox.ini
+++ b/tox.ini
@@ -13,7 +13,6 @@
     bin/write-mime-multipart
 shfiles =
     bin/cloud-localds
-    bin/cloud-publish-ubuntu
     bin/growpart
     bin/mount-image-callback
     bin/resize-part-image
